babel/core.py

```diff
@@ -562,6 +582,24 @@ def variants(self) -> localedata.LocaleDataDict:
         """
         return self._data['variants']
 
+    @property
+    def modifiers(self) -> localedata.LocaleDataDict:
+        """Identity mapping of modifiers with "@" prefixed (Temporary implementation)
+
+        TODO: This is not yet implemented, as it would need modification of the
+        locale_data files, so instead it just returns the key for now, with "@"
+        prefixed.
+
+        >>> Locale('de', 'DE').modifiers['euro']
+        u'@euro'
+        """
+
+        class IdentityDict(localedata.LocaleDataDict):
+            def __getitem__(self, key: str) -> Any:
+                return f'@{key!s}'
+
+        return IdentityDict(self._data)
+
```

Comment:
I'm not sure this needs to be implemented at all yet?

Reply:
Yeah, neither. I figure it doesn't hurt and provides a bit of an anchor to keep working on this, but the truth is that we probably will need to either remove it, or tap into the creation of `*.dat` files.

Reply:
Let's go with removing it for now, please? That way we can come up with a proper API later and aren't stuck freezing this as a public API :)